bcda/auth/ssas_test.go

```diff
@@ -151,7 +149,9 @@ func (s *SSASPluginTestSuite) TestVerifyToken() {
 		}
 
 		if clientId == os.Getenv("BCDA_SSAS_CLIENT_ID") && secret == os.Getenv("BCDA_SSAS_SECRET") {
-			b, _ := json.Marshal(struct{ Active bool `json:"active"` }{Active: true})
+			b, _ := json.Marshal(struct {
```

Comment:
This is good.  Can you also fill out a test for `SSASPlugin.ResetSecret()` here?

Reply:
done!